src/db/spanner/models.rs

```diff
@@ -193,7 +193,9 @@ impl SpannerDb {
         // Begin a transaction
         self.begin(false)?;
 
-        let user_id = params.user_id.legacy_id as u32;
+        let fxa_kid = params.user_id.fxa_kid;
+        let legacy = params.user_id.legacy_id as i32;
```

Comment:
we should be using fxa_uid everywhere in here now instead of mysql's legacy_id

Reply:
Ah! ok, I was confused about that part. thanks!